bin/echo.c

```diff
@@ -175,9 +180,21 @@ int print_connection_info(struct s2n_connection *conn)
         printf("Application protocol: %s\n", s2n_get_application_protocol(conn));
     }
 
-    printf("Curve: %s\n", s2n_connection_get_curve(conn));
-    printf("KEM: %s\n", s2n_connection_get_kem_name(conn));
-    printf("KEM Group: %s\n", s2n_connection_get_kem_group_name(conn));
+    const char *kem = s2n_connection_get_kem_name(conn);
+    if (strcmp(kem, "NONE") != 0) {
+        printf("Legacy TLS1.2 KEM: %s (%s, but the use of PQ key exchange with "
+               "TLS1.2 was experimental and is neither stable nor on a path to "
+               "standardization. The TLS1.3 version should be used instead: "
+               "https://github.com/aws/s2n-tls/blob/main/docs/usage-guide/topics/ch15-post-quantum.md)\n",
```

Comment:
Nit: You can link to the actual usage guide website if you want
```suggestion
               "https://aws.github.io/s2n-tls/usage-guide/ch15-post-quantum.html\n",
```

Reply:
Ooooo I do want! You dropped the final ")" tho so I couldn't just commit that suggestion.